test/Feature/MakeSymbolicAPI.c

```diff
@@ -25,4 +25,6 @@ int main() {
 
   klee_make_symbolic(&c);
   //CHECK-ERR-DAG: KLEE: ERROR: {{.*}} illegal number of arguments to klee_make_symbolic(void*, size_t, char*)
+
+  int i0 = klee_int((void*)0);
```

Comment:
@251, I would also add here a check that we print `renamed empty name to "unnamed"`, as above.